doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -719,6 +719,8 @@ bash configure --openjdk-target=${OPENJ9_CC_PREFIX} \
 
 :pencil: **DDR support:** You can build DDR support only on AArch64 Linux.  If you are building in a cross-compilation environment, you need the `--disable-ddr` option.
 
+:pencil: **CUDA support:** You can enable CUDA support if you are building on NVIDIA Jetson Developer Kit series.  Add `--enable-cuda --with-cuda=/usr/local/cuda --with-gdk=/usr/local/cuda` when you run `configure`.  The path `/usr/local/cuda` may be different depending on the version of JetPack.
```

Comment:
If the version of CUDA being used is 8.0 or newer, the `-with-gdk` option is not necessary.

Reply:
The CUDA version is 10.0, but I got an error on undefined `GDK_HOME` when I ran `configure` only with `--enable-cuda --with-cuda=/usr/local/cuda`.

Reply:
runtime/makelib/mkconstants.mk.ftl sets `GDK_HOME` for other platforms but does not for AArch64 Linux.
Can I assume the path `/usr/local/cuda` as the default path for AArch64 there?

Reply:
We should change `runtime/compiler/build/toolcfg/common.mk` to stop requiring `GDK_HOME` and only add it to the include path if it is set. With current versions of CUDA it is not required (`nvml.h` is part of the toolkit).

Reply:
See #9801.

Reply:
Updated removing `--with-gdk=/usr/local/cuda` from the document.
Waiting for #9801 to be merged.